ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Deploying an operator in the bundle format using operator-sdk run bundle command"
+
+cd /tmp
```

Comment:
Why is this needed?

Reply:
When run bundle is pulling the bundle image and extracting its contents, it would write those to ROOT, which would cause a permission denied error. So just changing the working directory to tmp instead.